src/modules/CRUD/services/attachments.js

```diff
@@ -0,0 +1,15 @@
+import Api from '@terralego/core/modules/Api';
+import { sanitizeCustomEndpoint } from './utils';
+
+const attachmentRequest = (endpoint, body, method = 'PUT') =>
+  Api.request(`${sanitizeCustomEndpoint(endpoint)}/`, { method, body });
+
+export const saveAttachment = attachmentRequest;
+
+export const deleteAttachment = endpoint =>
+  attachmentRequest(endpoint, {}, 'DELETE');
+
+export default {
+  saveAttachment,
+  deleteAttachment,
+};
```

Comment:
Why using named exports AND default export? Quite overkill…

*(all named exports may always be imported in a single object with `import * as attachmentMethods from './services/attachments.js';`)*

```suggestion
export const saveAttachment = attachmentRequest;
export const deleteAttachment = endpoint => attachmentRequest(endpoint, {}, 'DELETE');
```

Reply:
I don't why, i thought the linter does not accept a file without `export default`.  
It's now fixed. 👍